apps/mymedicare_cb/views.py

```diff
@@ -139,29 +153,29 @@ def generate_nonce(length=26):
 def mymedicare_login(request):
     redirect = settings.MEDICARE_REDIRECT_URI
     mymedicare_login_url = settings.MEDICARE_LOGIN_URI
-    redirect = req.pathname2url(redirect)
+    redirect = urllib_request.pathname2url(redirect)
     state = generate_nonce()
-    state = req.pathname2url(state)
+    state = urllib_request.pathname2url(state)
     request.session['state'] = state
     mymedicare_login_url = "%s&state=%s&redirect_uri=%s" % (
         mymedicare_login_url, state, redirect)
     next_uri = request.GET.get('next', "")
-    if request.user.is_authenticated():
-        return HttpResponseRedirect(next_uri)
```

Comment:
I thought this would fix CBBP 1013, but I guess it was insufficient. Still the opposite of what we want.

Reply:
Yes, but it turns out that, after discussion with Kelly, this is the opposite of what actually matches our user's needs. The situation in which a user gives access to multiple apps seems incredibly unlikely and uncommon. On the other hand, there are multiple situations in which it is likely that multiple users will give access from a single device. Specifically, we have couples and shared devices in a doctor's office or medical center.

Reply:
OK have we confirmed that `accounts.mymedicare.gov` is setup to do the other half of this? meaning if they're already login they let them login vs automatically redirecting back to us?

Reply:
They at least don't for sandbox accounts. I haven't checked on prod accounts for obvious reasons.